SR2021/2020-12-20-srawn-01.md

```diff
@@ -0,0 +1,47 @@
+- Read historical issues on the [SR(A)WN Archive](https://studentrobotics.org/srawn)
+
+### Holiday Social
+On the _24th December 17:00 - 20:00 UTC_ we will be having a festive social. We'll likely be playing some games (Among Us, Jackbox etc.) but nothing has been decided yet so shout if there's something you really want to do. The Google Meet link will be in [`#social`](https://studentrobotics.slack.com/archives/CF94T25GU) on Slack if you want to join!
+
+### Discord
+We're experimenting with using Discord this year as opposed to the forums. If you'd like to join and help answer questions or just join in the banter with the teams, the sign up link is [pinned](https://studentrobotics.slack.com/archives/C0EEJ0ASY/p1607552491072100) in `#general` on Slack.
+
+## Competition
+
+### The Game
+Our game this year, [Radars of the Lost Ark](https://studentrobotics.org/docs/rules/), challenges teams to score points by claiming territories using radios. Teams must navigate the arena scanning for tower transmissions and sending their own transmissions back to claim territories. Teams are awarded points at the end of the match for the maximum extent of their territory at any one time.
+
+### Game Modules
+As this competition year doesn’t feature a physical competition event, the structure of the competition has changed. The league matches will take place on four weekends throughout the year. After each league session the game will gain additional modules that will present new and different challenges to the competitors. If you have any cool ideas for modules please share them in `#game` on Slack or come along to a Competition Meeting.
+
+### Competition Meetings
+We have weekly Competition meetings on Thursdays at 8:30, these are open to all so feel free to drop in without notice or on an ad-hoc basis. These meetings, and any others, can be found on the [volunteer calendar](https://studentrobotics.org/runbook/volunteering/calendars/).
+
+### Friendlies Matches for Teams
+We're looking to organise friendly matches for the teams to have a chance to test their code in a less stressful environment. A chance for competitors to face off against other teams, socialize, and get some support from blueshirts. These will be happening 1 week before each league session at 10:30.
+
+### Webots Update
+
+Webots, the simulation software behind the competition simulator, released version 2021a with breaking changes. [A fix](https://github.com/srobo/competition-simulator/pull/251) has been submitted to the simulator so that 2020b and 2021a should both work.
+
+## Kit
+
+### Brain Updates
+
+The Brain Team is re-developing SR kit software for the SR2022 cycle.
+
+- We have started working on replacements for the legacy software currently used with the SR kit.
+- We now have developed an MVP replacement for herdsman called [Astoria](https://github.com/srobo/astoria), which uses asynchronous Python and JSON over MQTT.
```

Comment:
It might be worth saying what heardsman is (I gather this is the team code launcher)

Reply:
2e7836e